### PR TITLE
swift-environment-variable-injector 0.1.0

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Hey guys :))
It is failing but I can't see the details (gives me 404) to see what's happening. I'm able to run `bitrise run test` and `bitrise run audit-this-step` locally but seems like something is wrong. Any suggestions on how I can see the details? 
Thank's in advance :)

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+title: Swift Environment Variable Injector
+summary: |
+  Step to inject static values from environment variables into Swift code files before compilation and Continuous deployment allowing us to not expose our production keys and values in statically in our code and store sensitive values  safe.
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: extra space at the end of the line: `values  safe.`

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+description: |
+  TThis step allows us to inject CI pipelines environment variables values into Swift code  static values before compilation and Continuous deployment to a specific environment(Development, Testing or Production) where we can define the values injected for each build and deployment e.g. a API url that the App uses and is different for each environment. Also it allows us to not expose our production keys and values in statically in our code.
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: typo: `TThis `
extra space: `into Swift code  static values`

Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: you can use markdown format, to make the description more readable, if you want to.

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+source:
+  git: https://github.com/LucianoPAlmeida/bitrise-step-swift-envirionment-variable-injector.git
+  commit: 09112ef905f97aa5ea6a4c68722cae2e023ee4fe
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+type_tags:
+- dependency
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: dependency type steps should handle project dependencies, like cocoapods-install, carthage, ...
i would recommend to use: `utility` type for this step.

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: does the step use all of these dependencies?
if not it takes unnecessary extra time to install them before the step run.

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    title: Environment Variable Names to ignore
+  vars_to_ignore: 
+- opts:
+    description: |
+      All injectictions variables and values logged. Also the output files content.
+    is_expand: true
+    is_required: false
+    summary: Indicates if the injection should print logs.
+    title: Verbose mode
+    value_options:
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: `value_options` should have string values, like: `"true" and "false"`

File: steps/swift-environment-variable-injector/0.1.0/step.yml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    - true
+    - false
+  verbose: false
```
Comment (godrei) [inline steps/swift-environment-variable-injector/0.1.0/step.yml]: also default value should be a string: `"false"`

**Comment by LucianoPAlmeida** [comment]
@godrei Thank's for the feedback closing this to open a new one :))

**Pull request closed without merging**